### ISSUE TITLE
[ui5-list] A11y - Selected state of list item not visible when both Application + Windows high contrast settings enabled

### ISSUE THREAD

**Issue by Brahmansh** [SAP/ui5-webcomponents#4793]
### **Bug Description**
We've noticed when both Application + Windows high contrast settings enabled the selected state of list item is not visible.

### **Expected Behavior**
Selected state of list item should be shown and borders should be present to distinguish between each list item

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/List/?sap-ui-theme=sap_belize_hcb&isrtl=false. We can notice that in high contrast mode selected state of list item is visible correctly with different color e.g. blue / orange
2. But now enable windows high contrast settings following these steps - Settings --> Ease of Access --> High Contrast
3. We can notice that no background color for selected list item.

### **Context**
- UI5 Web Components version: 1.1.2
- OS/Platform: Windows
- Browser: All
- Affected component: ui5-list

### **Log Output / Stack Trace / Screenshots**

Only application high contrast enabled -
<img width="1363" alt="Screen Shot 2022-02-21 at 2 48 45 PM" src="https://user-images.githubusercontent.com/8809747/155038630-0613dd2d-9d34-4d26-a948-f21c3547306a.png">

Both Application + Windows high contrast enabled -
<img width="1366" alt="Screen Shot 2022-02-21 at 3 36 05 PM" src="https://user-images.githubusercontent.com/8809747/155038655-cf0361c2-9913-44e1-baa2-d2e3a4cc18f4.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by dobrinyonkov** [comment]
Hey @SAP/ui5-webcomponents-topic-p,

our styles seems to be overriden by Windows HCB theme. I did some research and find that there is a css property forced-color-adjust with which we can disable the OS overrides. Still, only supported by Chromium (Edge, Chrome). 

https://developer.mozilla.org/en-US/docs/Web/CSS/forced-color-adjust

We may need to plan a BLI and research on how to disabled the OS overrides.

**Comment by ilhan007** [comment]
Hello @Brahmansh it turns out the the usage of **"forced-color-adjust"** is the official solution for this scenario.

Can you try setting the following style:
```css
forced-color-adjust: none
```
to your body or top-most container, wrapping all UI5 Web Components that you use and check if it resolves the issue.

**Comment by Brahmansh** [comment]
Hi @ilhan007 , We had tried this earlier and it works fine. Only concern is that it is only supported by Chrome, Edge & IE. Not by FF / Safari.
https://developer.mozilla.org/en-US/docs/Web/CSS/forced-color-adjust

**Comment by ilhan007** [comment]
Hello @Brahmansh 

unfortunately we don't have a solution for FF/Safari (I think Safari is not that relevant for Windows, no?). 
For both [SAPUI5 controls](https://openui5nightly.hana.ondemand.com/controls) and [UI5 Web Components ](https://sap.github.io/ui5-webcomponents/)we prepare a official note that would become part of the documentation to state this limitation:

"Have in mind that UI5 Web Components/SAPUI5 controls are not optimised for the high contrast mode of Windows.
If you are using browsers other than Chrome and Edge & IE and if you have enabled both the Windows contrast setting and the High Contrast Theme, , this may cause conflicts and deficiencies in the theme can occur. In this case, please switch off the Windows setting."

BR,
ilhan
